src/pages/Admin/Admin.jsx

```diff
@@ -1,8 +1,58 @@
-import React from 'react';
+import React, { useState } from 'react';
+import List from './components/List/List';
+import Dashboard from './components/Dashboard/Dashboard';
+import Post from './components/Post/Post';
 import './Admin.scss';
 
 const Admin = () => {
-  return <div>admin</div>;
+  const [selectedMenu, setSelectedMenu] = useState('list');
+  const [isAddButtonClicked, setIsAddButtonClicked] = useState(false);
+
+  const handleMenuClick = menu => {
+    setSelectedMenu(menu);
+    setIsAddButtonClicked(false);
+  };
+
+  const handleAddButtonClick = () => {
+    setIsAddButtonClicked(true);
+  };
+
+  return (
+    <div className="admin">
+      <div className="navArea">
+        <h2 className="navTitle">관리자페이지</h2>
+        <ul className="navList">
+          <li>
+            <button
+              type="button"
+              onClick={() => handleMenuClick('list')}
+              className={`navMenu ${selectedMenu === 'list' ? 'active' : ''}`}
+            >
+              리스트
+            </button>
+          </li>
+          <li>
+            <button
+              type="button"
+              onClick={() => handleMenuClick('dashboard')}
+              className={`navMenu ${
+                selectedMenu === 'dashboard' ? 'active' : ''
+              }`}
+            >
+              대시보드
+            </button>
+          </li>
```

Comment:
죄측 메뉴 데이터를 변수로 관리하고 여기서 맵 메서드를 활용하는 게 좋아보입니다.
예를 들면,
```suggestion
          {ADMIN_MENUS.map(({ menu }) => (
            <li key={menu}>
              <button
                type="button"
                onClick={() => handleMenuClick(menu)}
                className={`navMenu${selectedMenu === menu ? ' active' : ''}`}
              >
                {menu}
              </button>
            </li>
          )}
```

그리고 `리스트`페이지일 때 add 버튼을 클릭했고, Post 컴포넌트가 출력되지만 `리스트` 버튼을 활성화하고 싶다면.. 많이 별로지만..
```suggestion
          {ADMIN_MENUS.map(({ menu }) => {
            const isCurrentMenu = (selectedMenu === menu) || (selectedMenu === '추가' && menu === '리스트');
          
            return (
              <li key={menu}>
                <button
                  type="button"
                  onClick={() => handleMenuClick(menu)}
                  className={`navMenu${isCurrentMenu ? ' active' : ''}`}
                >
                  {menu}
                </button>
              </li>
            )}}
```